includes/spoolchanges.js

```diff
@@ -29,29 +28,24 @@ const { pipeline } = require('node:stream/promises');
  * @param {number} bufferSize - the number of changes per batch/log line
```

Comment:
On line 26, type should be changed to `{object}`.

Reply:
966cdb5b159ed1866ae7ea7abc6ea2fdb25612e7